src/styles/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__title {
+    width: 100%;
+    margin: 40px 0 0;
+
+    font-weight: 500;
+    line-height: 150%;
+
+    color: $main-accent;
+  }
+
+  &__code {
+    width: 100%;
+    margin: 4px 0 0;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 140%;
+    text-align: start;
+    color: $secondary-accent;
+  }
+
+  &__review {
+    margin-top: 16px;
+    width: 100%;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__reviews-count {
+    margin: 0;
+
+    color: $main-accent;
+    font-size: 10px;
+    font-weight: 400;
+  }
+
+  &__price {
+    margin-top: 24px;
+    width: 100%;
+    height: 18px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-title {
+      margin: 0;
```

Comment:
reset file
check all over the document, if you reset default styles of element use separate file for it